scripts/dataset/main.js

```diff
@@ -0,0 +1,26 @@
+import scheduler from 'node-schedule';
+
+import logger from './logger/index.js';
+
+import { release } from './index.js';
+
+(async () => {
+  const args = process.argv.slice(2);
+  const publicationEnabled = args.includes('--publish');
+  const removeLocalCopyEnabled = args.includes('--remove-local-copy');
+  const scheduleEnable = args.includes('--schedule');
+  const argsWithoutOptions = args.filter(arg => !arg.startsWith('--'));
+  const [fileName] = argsWithoutOptions;
+
+  if (removeLocalCopyEnabled && !publicationEnabled) {
+    logger.warn('--remove-local-copy is only available with --publish option');
```

Comment:
shouldn't we throw here ? it's unclear what will be the the resulting behavior
I believe it will not remove local copy then? or even not create it?